certs/server-cert.pem

```diff
@@ -0,0 +1,78 @@
+        Issuer: C=US, ST=San Francisco, L=SOMA, O=Gravitational, CN=localhost
+        Validity
+            Not Before: Jan 16 07:13:34 2017 GMT
+            Not After : Jan 14 07:13:34 2027 GMT
+        Subject: O=Gravitational, CN=localhost
+        Subject Public Key Info:
+            Public Key Algorithm: rsaEncryption
+                Public-Key: (2048 bit)
+                Modulus:
+                    00:b3:64:4b:8e:5f:47:5e:24:ea:99:71:c0:dc:57:
+                    5b:d3:e7:94:a0:e0:5c:63:8d:42:15:cd:46:f8:7a:
+                    5a:31:73:0b:af:26:5d:e8:65:4f:17:70:8b:66:e8:
+                    b3:5d:f9:f3:14:39:31:a6:48:27:10:ae:22:d4:41:
+                    a5:a6:5e:5d:6f:76:78:a5:93:d6:e3:61:aa:60:27:
+                    5f:23:60:3e:c2:3a:fb:10:c7:8b:3f:8a:8e:fb:11:
+                    d2:c4:1a:b0:bc:35:3d:a1:04:6a:72:78:44:ab:f6:
+                    13:44:2a:60:41:01:34:99:b2:cf:6f:23:1e:ea:54:
+                    29:b8:32:01:5d:9e:91:bf:70:8c:64:6e:b6:49:0d:
+                    d9:ed:7c:26:34:ef:3e:7d:f1:37:cd:63:f7:c8:34:
+                    90:85:5f:aa:cc:12:b7:fa:6e:29:a9:95:97:9c:f0:
+                    4e:6f:38:16:d4:e3:87:a7:cb:3d:7e:37:00:de:ff:
+                    86:fa:7d:60:b2:e9:b5:0c:46:66:93:01:97:91:3d:
+                    f8:bc:0c:a1:f1:66:39:c3:58:83:92:85:8e:cc:f6:
+                    1d:cf:49:2e:8d:92:94:bc:9a:4a:1b:1d:9d:6e:57:
+                    c8:c0:23:57:3c:f2:a7:57:5d:5a:fd:72:49:6a:32:
+                    f9:0d:00:38:b3:94:23:50:0b:21:8b:01:cd:9b:fb:
+                    49:d9
+                Exponent: 65537 (0x10001)
+        X509v3 extensions:
+            X509v3 Basic Constraints: 
+                CA:FALSE
+            X509v3 Extended Key Usage: 
+                TLS Web Client Authentication, TLS Web Server Authentication
+            X509v3 Key Usage: 
+                Digital Signature, Key Encipherment
+            X509v3 Subject Alternative Name: 
+                IP Address:127.0.0.1
+    Signature Algorithm: sha512WithRSAEncryption
+         35:a6:75:82:4b:a4:cb:08:24:cb:a0:ef:7a:17:1b:3d:af:2b:
+         41:75:03:b6:d5:34:c0:48:00:e7:47:c4:1e:e0:2b:7f:72:c9:
+         5f:3e:34:66:ff:33:f3:6f:db:7f:33:d4:7e:5f:83:59:bd:6a:
+         f4:25:d9:f2:de:ea:39:9c:02:09:17:37:24:ac:a1:fb:a9:bc:
+         ed:c1:de:01:e2:6d:c5:66:81:af:0c:1a:85:d4:fa:fe:9f:2b:
+         3c:3b:40:2f:68:b4:a5:56:79:59:6d:64:97:95:f1:4b:35:dd:
+         83:f5:c7:58:7c:50:7d:ba:b7:8e:93:a0:94:70:38:9a:96:14:
+         7b:a3:db:e2:20:44:47:f6:13:6f:e1:4d:6e:20:e1:c5:0e:38:
+         76:48:b9:f8:9f:e8:d2:28:5f:ca:e7:06:5c:8a:e0:cb:78:ac:
+         12:df:ba:9e:39:28:27:b2:c5:a7:a2:ce:8a:47:0e:e4:ed:61:
+         bb:59:8e:a9:27:49:80:d7:32:fb:8f:18:6b:9b:00:0e:4e:51:
+         f7:08:e8:c8:b3:63:8f:d1:62:c7:39:ff:11:94:ea:a7:0b:1f:
+         ab:34:a4:26:1d:d6:03:8c:4a:a5:c4:eb:47:84:7e:2d:29:ac:
+         e8:6c:d1:77:15:76:88:d6:19:c8:ef:e4:8d:5f:b4:0e:ed:1e:
+         8d:5c:c3:67
```

Comment:
nit: remove all this decoded metadata, keep only the PEM block

Reply:
This is the PEM file copied directly from the given files from the full stack challenge. Do you want me to remove the metadata still or doesn't matter?

Reply:
yes, remove the metadata please.
@alex-kovoy do you know where this file originates from? we should clean that up too in our challenge files.

Reply:
@awly 
https://github.com/gravitational/fakeiot/tree/master/fixtures